app/assets/stylesheets/organisms/_form-card.scss

```diff
@@ -120,3 +120,82 @@
     }
   }
 }
+
+.form-card__target {
+  text-align: center;
+  border: solid 5px transparent;
+  font-weight: bold;
+  color: $color-grey;
+  padding: 3rem;
+  margin-bottom: 3rem;
+  min-height: 5rem;
+  cursor: pointer;
+
+  &.dz-drag-hover {
+    border: dashed 5px $color-light-grey;
+  }
+
+  i.icon-file_upload {
+
+  }
+}
+
+.form-card__documents {
+  &:empty { display: none; }
+
+  .dz-preview {
```

Comment:
It stands for `dropzone`. What's Dropzone? [Dropzone.js] is the OSS JS library that shubox wraps around to get a lot of the upload plumbing to work. It's pretty dope.

[Dropzone.js]: http://www.dropzonejs.com/

Instead of painting over it w/the shubox libs and namespacing it on my own I've just let it be, let it maintain it's rightful identity.